src/theme/components/checkbox.ts

```diff
@@ -0,0 +1,79 @@
+import { checkboxAnatomy } from "@chakra-ui/anatomy";
+import { createMultiStyleConfigHelpers, defineStyle } from "@chakra-ui/react";
+
+const { definePartsStyle, defineMultiStyleConfig } =
+  createMultiStyleConfigHelpers(checkboxAnatomy.keys);
+
+const baseStyleControl = defineStyle(() => {
+  return {
+    borderRadius: "2px", // change the border radius of the control
+    border: "2px solid",
+    borderColor: "gray.700",
+    flexShrink: 0,
```

Comment:
I removed this and it seemed to still function as expected? I don't see this being set in Chakra's default theme so it may be unnecessary. If it is, we should remove it to make these components as unopinionated as possible.